src/rgw/rgw_op.cc

```diff
@@ -2084,6 +2086,20 @@ int RGWGetObj::get_data_cb(bufferlist& bl, off_t bl_ofs, off_t bl_len)
   return send_response_data(bl, bl_ofs, bl_len);
 }
 
+int RGWGetObj::get_lua_filter(std::unique_ptr<RGWGetObj_Filter>* filter, RGWGetObj_Filter* cb) {
+  std::string script;
+  const auto rc = rgw::lua::read_script(s, store, s->bucket_tenant, s->yield, rgw::lua::context::getData, script);
```

Comment:
this broke the build on main. it looks like a conflict with a1e21d0886c5c3ce5e65819aa12cf807f51e2061 which changed the signature of `read_script()`

Reply:
it's not clear how we're supposed to get that LuaManager all the way into RGWGetObj/PutObj. in the meantime, i prepared a revert at https://github.com/ceph/ceph/pull/47612